lib/node/runner/checker.go

```diff
@@ -614,44 +614,29 @@ func finishOperation(st *storage.LevelDBBackend, source string, op operation.Ope
 	}
 }
 
-func finishCreateAccount(st *storage.LevelDBBackend, source string, op operation.CreateAccount, log logging.Logger) (err error) {
-
-	var baSource, baTarget *block.BlockAccount
-	if baSource, err = block.GetBlockAccount(st, source); err != nil {
-		err = errors.ErrorBlockAccountDoesNotExists
-		return
-	}
-	if baTarget, err = block.GetBlockAccount(st, op.TargetAddress()); err == nil {
-		err = errors.ErrorBlockAccountAlreadyExists
-		return
-	} else {
-		err = nil
-	}
-
-	baTarget = block.NewBlockAccountLinked(
+func finishCreateAccount(st *storage.LevelDBBackend, source string, op operation.CreateAccount, log logging.Logger) error {
+	baTarget := block.NewBlockAccountLinked(
 		op.TargetAddress(),
 		op.GetAmount(),
 		op.Linked,
 	)
-	if err = baTarget.Save(st); err != nil {
-		return
+	if err := baTarget.Save(st); err != nil {
+		return err
```

Comment:
Like `finishPayment`, this also just can make `panic`, is there any reason to return error?

Reply:
Since it writes to LevelDB, there are much more potential for failure, that's why it's treated differently.

Reply:
I agreed.
finishXXX should not return an error.
How about using `MustXXX` pattern?